nixos/modules/services/networking/zerotierone.nix

```
@@ -83,6 +96,14 @@ in
           ln -s ${localConfFile} ${localConfFilePath}
         '';
 
+      postStart =
+        lib.optionalString (cfg.generateSystemdNetworkdConfig && cfg.systemdManagerPackage != null)
```

lib.mkPackageOption is non-nullable by default.

```suggestion
        lib.optionalString cfg.generateSystemdNetworkdConfig
```